apps/designer/app/shared/nano-states/nano-states.ts

```diff
@@ -23,6 +23,16 @@ export const useSetPresetStyles = (presetStyles?: PresetStyles) => {
   });
 };
 
+export const stylesContainer = createValueContainer<Styles>([]);
+export const useStyles = () => useValue(stylesContainer);
+export const useSetStyles = (styles?: Styles) => {
+  useState(() => {
```

Comment:
this is where I did the research last time and ended up reverting the usage of useState for things like this, because its a side effect, and ended up with creating this hook useSyncInitializeOnce